test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -267,6 +265,10 @@ public void execute_tryToViewMissingPerson_errorMessage() throws Exception {
         expectedAB.addPerson(p2);
 
         addressBook.addPerson(p2);
+        // This statement is necessary because view command does not trigger saving changes after executed.
```

Comment:
Consider abstracting this away in a generator/constructor made specially for commands that do not trigger saves.